addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -96,7 +96,7 @@ def _check_move_configuration(self, move):
                 error_message.append(_("- Transport document number and date is required when Transportation Mode is Rail,Air or Ship"))
         if error_message:
             error_message.insert(0, _("The following information are missing on the invoice (see eWayBill tab):"))
-        goods_lines = move.invoice_line_ids.filtered(lambda line: not (line.display_type or line.is_rounding_line or line.product_id.type == "service"))
+        goods_lines = move.invoice_line_ids.filtered(lambda line: not (line.display_type in ('line_section', 'line_note', 'rounding') or line.product_id.type == "service"))
```

Comment:
```suggestion
        goods_lines = move.invoice_line_ids.filtered(lambda line: not (line.display_type != 'product' or line.product_id.type == "service"))
```

Reply:
@jco-odoo Before this, it was the same, so I suggest not improving in the current one.
https://github.com/odoo/odoo/commit/c81595db6a7a135a3289751a9b881b2f947bc6d2#diff-fe059f98e5cefbc09ee21ab8b14d90ba3676d8b0998d9589865d1b5a2d708a7eL109

Reply:
There are no rounding lines in `invoice_line_ids`
Only notes, sections and products.
```suggestion
        goods_lines = move.invoice_line_ids.filtered(lambda line: line.display_type == 'product' and line.product_id.type != "service")
```

Reply:
Definitely we Improve this in master